base_limiter.go

```diff
@@ -34,7 +34,7 @@ func NewBaseLimiter(
 	targetExtensionsMap := make(map[string]struct{}, len(targetExtensions))
 	if len(targetExtensions) > 0 {
 		for _, ext := range targetExtensions {
-			if ext[0] != '.' {
+			if len(ext) > 0 && ext[0] != '.' {
 				ext = "." + ext
 			}
 			targetExtensionsMap[strings.ToLower(ext)] = struct{}{}
```

Comment:
It seems better to exclude illegal ext.
https://go.dev/play/p/tAFv_4iuCHr

Reply:
Are you saying that having a file extension as "" (empty string) is an abnormal case? I thought that in the context of the web, there are cases where files don't have extensions.

Reply:
https://go.dev/play/p/_8Znd6BTTDv

Reply:
I think it's a confusing behavior that the root path is limited to the target file with no file extension.
How about this?

https://github.com/2manymws/rlutils/blob/49d6079f9ac4e30a15aba4df45d1c42bb64711db/base_limiter.go#L68C44-L68C54

Reply:
Might as well change isTargetExtensions as a fix to https://github.com/2manymws/rlutils/pull/9/files#r1436334082